src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (Figure figure : figures) {
+            figure.draw();

```

Comment:
1) You can do it within one for-loop
2) Task complete incorrectly, pay attention to the task description